aries-site/src/examples/components/fileinput/FileInputConfirmRemoveExample.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import { Box, Button, FileInput, Heading, Layer, Paragraph } from 'grommet';
-import { ButtonGroup } from '../../templates/ButtonGroup';
+import { ButtonGroup } from 'aries-core';
```

Comment:
This is a broader question but do we have any flexibility to rename this directory to something like `hpe-design-system-templates` or something more true to today since we don't use the term "aries" anymore?

Reply:
I would think so.